capfile/src/test/java/pro/javacard/sdk/TestSDKCompatibilityTable.java

```diff
@@ -0,0 +1,110 @@
+        Assert.assertTrue(SDKCompatibilityTable.isJDKCompatible(SDKVersion.V310, 11));
+        Assert.assertTrue(SDKCompatibilityTable.isJDKCompatible(SDKVersion.V310, 17));
+        Assert.assertFalse(SDKCompatibilityTable.isJDKCompatible(SDKVersion.V310, 21));
+        
+        // Test JDK compatibility for 3.2.0 variants
+        Assert.assertTrue(SDKCompatibilityTable.isJDKCompatible(SDKVersion.V320_24_1, 11));
+        Assert.assertTrue(SDKCompatibilityTable.isJDKCompatible(SDKVersion.V320_24_1, 17));
+        Assert.assertFalse(SDKCompatibilityTable.isJDKCompatible(SDKVersion.V320_24_1, 8));
+        
+        // Test V320_25_0 supports newer JDK versions
+        Assert.assertTrue(SDKCompatibilityTable.isJDKCompatible(SDKVersion.V320_25_0, 21));
+    }
+
+    @Test
+    public void testMultitargeting() {
+        // Test that only V310 and V32x versions support multitargeting
+        Assert.assertFalse(SDKCompatibilityTable.isMultitarget(SDKVersion.V222));
+        Assert.assertFalse(SDKCompatibilityTable.isMultitarget(SDKVersion.V304));
+        Assert.assertTrue(SDKCompatibilityTable.isMultitarget(SDKVersion.V310));
+        Assert.assertTrue(SDKCompatibilityTable.isMultitarget(SDKVersion.V320));
+        Assert.assertTrue(SDKCompatibilityTable.isMultitarget(SDKVersion.V320_24_1));
+        Assert.assertTrue(SDKCompatibilityTable.isMultitarget(SDKVersion.V320_25_0));
+    }
+
+    @Test 
+    public void testTargeting() {
+        // Test that V310 can target appropriate versions
+        Assert.assertTrue(SDKCompatibilityTable.canTarget(SDKVersion.V310, SDKVersion.V304));
+        Assert.assertTrue(SDKCompatibilityTable.canTarget(SDKVersion.V310, SDKVersion.V305));
+        Assert.assertTrue(SDKCompatibilityTable.canTarget(SDKVersion.V310, SDKVersion.V310));
+        Assert.assertTrue(SDKCompatibilityTable.canTarget(SDKVersion.V310, SDKVersion.V320));
+        
+        // Test that V320 variants can target appropriate versions
+        Assert.assertTrue(SDKCompatibilityTable.canTarget(SDKVersion.V320, SDKVersion.V304));
+        Assert.assertTrue(SDKCompatibilityTable.canTarget(SDKVersion.V320_24_1, SDKVersion.V305));
+        
+        // Test that older versions cannot target newer versions
+        Assert.assertFalse(SDKCompatibilityTable.canTarget(SDKVersion.V304, SDKVersion.V310));
+        Assert.assertFalse(SDKCompatibilityTable.canTarget(SDKVersion.V222, SDKVersion.V304));
+    }
+
+    @Test
+    public void testJDKCompatibilityDescription() {
+        // Test JDK compatibility descriptions
+        Assert.assertEquals(SDKCompatibilityTable.getJDKCompatibilityDescription(SDKVersion.V222), "JDK 1-8");
+        Assert.assertEquals(SDKCompatibilityTable.getJDKCompatibilityDescription(SDKVersion.V304), "JDK 1-11");
+        Assert.assertEquals(SDKCompatibilityTable.getJDKCompatibilityDescription(SDKVersion.V320_25_0), "JDK 8+");
+    }
+
+    @Test
+    public void testMinMaxJDKVersions() {
```

Comment:
rather have a list of JDK-s (and approximate versions to nearest LTS when detecting)